README.md

```diff
@@ -108,7 +108,7 @@ For general understanding of how prebuilt UI components of Adyen work you can fo
 
 ### Configuration
 
-To read more about other configuration, see [Full list](docs\Configuration.md).
+To read more about other configuration, see [full list](configuration).
```

Comment:
Suggestion: "see the [full list](configuration)"

Reply:
Fixed [5aecb70](https://github.com/Adyen/adyen-react-native/pull/146/commits/5aecb7059029f427ac1ae67cfc209baf38c8cedf)